software/profiling/Vampir.rst

```diff
@@ -1,11 +1,320 @@
+Overview
+=========
+
+Vampir is a software performance visualizer focused on highly parallel applications. It presents a unified view on an application run
+including the use of programming paradigms like MPI, OpenMP, PThreads, CUDA, OpenCL and OpenACC.
+It also incorporates file I/O, hardware performance counters and other performance data sources.
+Various interactive displays offer detailed insight into the performance behavior of the analyzed application.
+Vampir's scalable analysis server and visualization engine enable interactive navigation of large amounts of performance data.
+Score-P and TAU generate OTF2 trace files for Vampir to visualize.
+
+
+Usage
+=======
+
+Vampir can be run a few different ways depending on a couple of factors. If you have a large trace file it
+would benefit from utilizing VampirServer to process the trace file with the systems compute power
+while reverse connected to the Vampir GUI. If the trace file is small enough, it would do just fine viewing on a
+login node or your local machine.
+
+.. _vamps:
+
+``Vampirserver`` is the backend software component that can run across multiple compute nodes taking advantage
+of the machines memory, this in turn provides a peak performance for viewing large trace files.
+
+.. Note:: Vampirserver does not take advantage of GPU components
+
+
+This page will go through the 3 different types of connections available for using Vampir on Summit.
+
+For logging onto Summit you will need to specify ``-X`` as part of your ``SSH`` command, this will
+activate the `x11 forwarding <https://docs.olcf.ornl.gov/connecting/index.html#x11-forwarding>`_ so you can launch a GUI on Summit. Please visit this link if you need more
+information for logging onto `Summit <https://docs.olcf.ornl.gov/systems/summit_user_guide.html#connecting>`_
+
+.. code::
+
+   $ ssh -X <USERID>@summit.olcf.ornl.gov
+
+
+Vampir on a Login Node
+======================
+
+.. Warning::
+
+   Running Vampir on a login node is best suited for small trace files.
+
+
+.. image:: /images/vampir_login_gui.png
+   :align: left
+   :width: 280px
+   :height: 125px
+
+|
+|
+|
+|
+|
+|
+
+After logging onto Summit, execute the series of commands below:
+
+.. code::
+
+   $ module load vampir
+
+   $ vampir &
+
+From here you can load a file resident on the file system.
+
+.. image:: /images/vampir_splash_screen_file_selection.png
+   :align: left
+   :width: 300px
+   :height: 200px
+
+|
+|
+|
+|
+|
+|
+|
+|
+|
+
+
+
+
+Vampir Using Vampirserver
+=========================
+
+
+.. Tip::
```

Comment:
Again, here I would just use a Note

Reply:
After speaking with Veronica, I added this Warning for the first Vampir option..."Do not run Vampir on a login node for trace files > 1 Gb! Please see the next 2 sections for running larger trace files."